src/content/structured/styles/elevation.mdx

```diff
@@ -33,19 +33,20 @@ There are three layers:
-| <span class="css-token">--ic-z-index-dialog</span>                             | Dialog                                                                                                     | Base + 100             |
-| <span class="css-token">--ic-z-index-tooltip</span>                            | [Tooltip](/components/tooltip)                                                                             | Base + 110             |
-| <span class="css-token">--ic-z-index-classification-banner</span>              | [Classification banner](/components/classification-banner)                                                 | Base + 200             |
+| <span style={{width: '5vw', display: 'inline-block'}}>**Token**</span> | **Component**                                                                                              | **Calculated z-index** |
+| ---------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------- | ---------------------- |
+| <span class="css-token">--ic-z-index-base-value</span>                 | Base z-index value. Used to calculate z-index of other ic components that sit above the base layer.        | 0 (Base)               |
+| <span class="css-token">--ic-z-index-page-header</span>                | [Page header](/components/page-header)                                                                     | Base + 10              |
+| <span class="css-token">--ic-z-index-back-to-top</span>                | [Back to top](/components/back-to-top)                                                                     | Base + 20              |
+| <span class="css-token">--ic-z-index-menu</span>                       | Menu for [select](/components/select) and [search bar](/components/search-bar)                             | Base + 50              |
+| <span class="css-token">--ic-z-index-popover</span>                    | Popover menu                                                                                               | Base + 50              |
+| <span class="css-token">--ic-z-index-navigation-item</span>            | Navigation item used in [top navigation](/components/top-nav) and [side navigation](/components/side-nav). | Base + 50              |
+| <span class="css-token">--ic-z-index-navigation-menu</span>            | Navigation menu used in [top navigation](/components/top-nav) at smaller breakpoints.                      | Base + 60              |
+| <span class="css-token">--ic-z-index-side-navigation</span>            | [Side navigation](/components/side-nav)                                                                    | Base + 60              |
+| <span class="css-token">--ic-z-index-dialog</span>                     | Dialog                                                                                                     | Base + 100             |
```

Comment:
Should be we displaying unavailable components on the z-index list? Same for popup menu? I guess popup menu might be alright because I envisage us updating the guidance site in the next release which will include popover menu

Reply:
removed dialog token